src/assets/slides/pydata-london-2026_files/libs/revealjs/plugin/reveal-menu/menu.css

```
@@ -0,0 +1,346 @@
+.slide-menu-wrapper {
+  font-family: 'Source Sans Pro', Helvetica, sans-serif;
+}
+
+.slide-menu-wrapper .slide-menu {
+  background-color: #333;
+  z-index: 200;
+  position: fixed;
+  top: 0;
+  width: 300px;
+  height: 100%;
+  /*overflow-y: scroll;*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)